specs/cli/commands/generate.md

```diff
@@ -0,0 +1,74 @@
+# FAST Generate (WIP)
+
+## Overview
+FAST `generate` is a command that is used to scaffold out the file structure of a new component in a FAST project.
+
```

Comment:
I think we might want to expand this over time, where we can generate different resources (not just custom elements). It might be worth taking that into account in this spec. Aurelia already does this with it's own `generate` command, requiring a `<resource>` be provided.

Reply:
Fair. That thought did occur to me as well. I noticed that about Aurelia's `generate <resource>` What are some of the constructs that come to your mind that will need to be generated for FAST?

Reply:
I could potentially see generators for behaviors, templates, styles, design tokens, color recipes, and probably other concepts as well. Not sure how useful some of those might be, but making elements a configuration of `generate` will allow us to grow organically as we find use pieces to template out.

Reply:
@nicholasrice Sounds good, I like that. I'll update the spec to require a resource type as an argument and add a section of possible future iterations.

Reply:
FAST CLI does exist today as well written in Bash but used for Azure management, which essentially is an Azure CLI wrapper for automating tear down and stand up of FAST infrastructure.  In the long term is maybe nice to merge the two. `fast cloud --arg-1 --arg-2` for example.  I'm not saying this should be in place now, only to keep it in the back of your mind.

Reply:
@awentzel Yes, I'm inclined to agree that these features can be added to future iterations of the CLI.